bin-src/tasks/upload.test.ts

```diff
@@ -90,6 +91,8 @@ describe('validateFiles', () => {
 });
 
 describe('traceChangedFiles', () => {
+  jest.spyOn(trimStatsFile, 'readStatsFile').mockReturnValueOnce(Promise.resolve(undefined));
```

Comment:
How was this file previously mocking the reading of JSON?

Reply:
It wasn't. Just resolved to undefined and did not break. The tests could be improved if there are use cases that require stats files to contain something though